### PR TITLE
Conversion of bsStyle and cleanup

### PR DESCRIPTION
**Pull request by sg00dwin**
Screen Shot 2019-09-30 at 3 18 55 PM

### CODE REVIEW

File: frontend/public/components/overview/build-overview.tsx [hunk 1]
```diff
@@ -110,7 +111,7 @@ const BuildOverviewList: React.SFC<BuildOverviewListProps> = ({ buildConfig }) =
           </div>
           {canStartBuild && (
             <div>
-              <Button bsStyle="default" bsSize="xs" onClick={onClick}>
+              <Button variant="link" isInline onClick={onClick}>
```
Comment (rhamilto) [inline frontend/public/components/overview/build-overview.tsx]: This was a regular button before, not a link.

File: frontend/public/style/_overrides.scss [hunk 2]
```diff
@@ -119,6 +119,13 @@ h6 {
   }
 }
 
+.pf-c-button.pf-m-inline.pf-m-link.co-dashboard-card__button-link {
```
Comment (rhamilto) [inline frontend/public/style/_overrides.scss]: You don't need `isInline` in the jsx and `padding: 0`.  Are all these declarations needed with the PF4 button styles?

I prefer keeping this code with the component (in frontend/public/components/dashboard/dashboard-card/card.scss).  I'd remove the extra specificity and add `!important` where needed.

File: frontend/public/components/overview/build-overview.tsx (hunk 1 above)
Comment (sg00dwin) [reply frontend/public/components/overview/build-overview.tsx]: updated

File: frontend/public/style/_overrides.scss (hunk 2 above)
Comment (sg00dwin) [reply frontend/public/style/_overrides.scss]: updated

**Pull request merged**